lib/gorgon/command.rb

```diff
@@ -0,0 +1,137 @@
+require "rubygems"
+require 'gorgon'
+require 'gorgon/originator'
+require 'gorgon/listener'
+require 'gorgon/rsync_daemon'
+require 'gorgon/worker_manager'
+require 'gorgon/ping_service'
+require 'gorgon/gem_service'
+require 'gorgon/version'
+require 'gorgon/listener_installer'
+require 'gorgon/settings/initial_files_creator'
+
+module Gorgon
+  class Command
+    attr_reader :argv
+    WELCOME_MESSAGE = "Welcome to Gorgon #{Gorgon::VERSION}"
+    USAGE = <<-EOT
+USAGE: gorgon <command> [<args>]
+
+COMMANDS:
+  start                       remotely run all tests specified in gorgon.json
+  listen                      start a listener process using the settings in gorgon_listener.json
+  ping                        ping listeners and show hosts and gorgon's version they are running
+  init [<framework>]          create initial files for current project
+  install_listener            run gorgon listener as a daemon process
+  start_rsync <directory>     start rsync daemon. Run this command in File Server
+  stop_rsync                  stop rsync daemon.
+  manage_workers
```

Comment:
You can remove the `manage_workers` from the usage documentation. That command is supposed to be used internally by a gorgon process in here: https://github.com/nulogy/Gorgon/blob/master/lib/gorgon/listener.rb#L154